.nvmrc

```diff
@@ -1 +1 @@
-lts/carbon
+8.12.0
```

Comment:
Is there a specific reason we're not switching to node v10?

Reply:
We're sticking with the LTS version for the sake of stability.